src/components/Calculator.js

```diff
@@ -1,15 +1,117 @@
-import React from 'react';
-import Button from './Button';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 import '../styles.css';
 
-function Calculator() {
+const Calculator = () => {
+  const [count, setCount] = useState({
+    total: 0,
+    operation: null,
+    next: null,
+  });
+
+  const clickHandler = (button) => {
+    const output = calculate(count, button);
+    setCount(output);
+  };
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍